Content.Client/Pinpointer/UI/NavMapControl.cs

```diff
@@ -367,15 +374,21 @@ protected override void Draw(DrawingHandleScreen handle)
         if (_beacons.Pressed)
         {
             var rectBuffer = new Vector2(5f, 3f);
+            var cache = IoCManager.Resolve<IResourceCache>();
+            Font font;
```

Comment:
Store the dependency rather than resolving the threadlocal every frame please.

Reply:
Ok I think I got it